crt_portal/crt_portal/urls.py

```diff
@@ -81,6 +81,7 @@
     path('privacy-policy', TemplateView.as_view(template_name="privacy.html"), name='privacy_policy'),
     path('hate-crime-human-trafficking', TemplateView.as_view(template_name="hate_crime_human_trafficking.html"), name='hate_crime_human_trafficking'),
     path('housing-resources', TemplateView.as_view(template_name="hce_resources.html"), name='hce_resources'),
+    path('voting-resources', TemplateView.as_view(template_name="vot_resources.html"), name='vot_resources'),
```

Comment:
We might consider renaming the template voting_resources.html for consistency.

Reply:
We are following the pattern from the hec_resources.  This is consistent with the previous pattern.